pendulum/utils/_compat.py

```diff
@@ -2,12 +2,20 @@
 
 import sys
 
+from typing import TYPE_CHECKING
+
 PYPY = hasattr(sys, "pypy_version_info")
 PY38 = sys.version_info[:2] >= (3, 8)
 
-try:
-    from backports import zoneinfo
-except ImportError:
-    import zoneinfo  # type: ignore[no-redef]
+if TYPE_CHECKING:
```

Comment:
will having only the `TYPE_CHECKING` part satisfy `mypy`? we are using `pyupgrade`, which understands this type of syntax (unlike try-ImportError) and handles it when upgrading.

Reply:
> will having only the `TYPE_CHECKING` part satisfy `mypy`? we are using `pyupgrade`, which understands this type of syntax (unlike try-ImportError) and handles it when upgrading.

If you are asking if the whole `if TYPE_CHECKING/else` block can be simplified to just the version check condition, the answer is yes. That whole block can be simplified to:

```python
if sys.version_info < (3, 9):
    from backports import zoneinfo
else:
    import zoneinfo
```

This will work at runtime and during type checking (in both `mypy` and `pyright` and I would suspect other type checkers as well). IMO the simplified version is the better solution, but I was trying to make as few changes to the runtime behavior as possible.

Reply:
Go with this solution, please. We are going into 3.0 anyway, so it's ok if there are some changes.

Reply:
I've updated PR to address this